microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -256,17 +272,28 @@ Microsoft Edge indicates when your precise location is being shared on the right
+If you turn off the **Allow access to location on this device** and **Allow apps to access your location** settings, some sites may still estimate your location using other technologies (such as Bluetooth, WiFi, cellular modem, or IP address) with varying degrees of accuracy. With Windows location settings disabled, precise location experiences that you allow in Microsoft Edge may also be inaccurate. For more information about Windows location settings, navigate to [Windows location service and privacy](https://support.microsoft.com/help/4468240).
 
-Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Microsoft location service, Microsoft Edge generates a new random ID for each request.
+Microsoft Edge doesn't store your geolocation coordinates. When making requests to the Windows location service, Microsoft Edge generates a new random ID for each request.
 
 **InPrivate** browsing uses the precise location permission setting of the profile from which the **InPrivate** session was launched. **Guest** mode always asks you for permission before granting the site your precise location.
 
 
+<!-- ====================================================================== -->
+## Image Descriptions
+When browsing the web in Microsoft Edge, screen reader users may encounter images that are not annotated with alt text. Without alt text to describe these images, they are effectively invisible to screen reader users. When Image Descriptions is turned on, Microsoft Edge will detect these images without alt text and send them to Azure Cognitive Services to generate captions. The generated captions are then announced to screen reader users. Even when Image Descriptions is turned on, images are only sent to the service when a screen reader (or other assistive technology) is connected to Microsoft Edge. 
+
+Only raw image data is sent to Azure Cognitive Services. Images are sent un-encrypted to reduce latency. No user identifiers are included in the requests to the service. Images are not stored or saved on Microsoft servers; Images are discarded immediately after the captions are generated. 
+
+To control Image Descriptions, navigate to edge://settings/accessibility. Toggle the **Get image descriptions from Microsoft for screen readers** setting. 
+
+Image Descriptions can be turned on for the current web page without enabling for all web pages. While using a screen reader, with focus on a web page element, open the contextual menu (right-click), select **Get image descriptions from Microsoft** > **Just once**. The current web page will be scanned for all images without alt text and send those to the service for descriptions. The Image Description feature will remain off for other pages and future browsing. 
```

Comment:
```suggestion
Image Descriptions can be turned on for the current web page without enabling for all web pages. While using a screen reader, right-click the webpage, select and then select **Get image descriptions from Microsoft** > **Just once**. The current web page will be scanned for all images without alt text and send those to the service for descriptions. The Image Description feature will remain off for other pages and future browsing. 
```

Reply:
the word 'contextual' is not in _Microsoft Writing Style Guide_.  
Applied _Edge Dev Style Guide_.

Reply:
There are 3 other instances of "contextual menu (right-click)" in the whitepaper today. Do you want me to update those as well?